app/main.py

```diff
@@ -188,5 +197,5 @@ def tap_source():
 
 if __name__ == '__main__':
     db.create_tables([Label, HasTapped])
-    HasTapped.create(has_tapped=0)
+    HasTapped.create(has_tapped=0, tap_successful=0)
```

Comment:
I suspect that `label.db` is deleted when the Interactive Label starts, but you might want to handle the following error incase it isn't. Else we'll need to manually delete `label.db` on all the interactive labels when they update to this code.

```bash
labelinteractive    | Traceback (most recent call last):
labelinteractive    |   File "/usr/local/lib/python3.7/runpy.py", line 193, in _run_module_as_main
labelinteractive    |     "__main__", mod_spec)
labelinteractive    |   File "/usr/local/lib/python3.7/runpy.py", line 85, in _run_code
labelinteractive    |     exec(code, run_globals)
labelinteractive    |   File "/code/app/main.py", line 200, in <module>
labelinteractive    |     HasTapped.create(has_tapped=0, tap_successful=0)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 6346, in create
labelinteractive    |     inst.save(force_insert=True)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 6556, in save
labelinteractive    |     pk = self.insert(**field_dict).execute()
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 1907, in inner
labelinteractive    |     return method(self, database, *args, **kwargs)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 1978, in execute
labelinteractive    |     return self._execute(database)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 2745, in _execute
labelinteractive    |     return super(Insert, self)._execute(database)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 2474, in _execute
labelinteractive    |     cursor = database.execute(self)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 3157, in execute
labelinteractive    |     return self.execute_sql(sql, params, commit=commit)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 3151, in execute_sql
labelinteractive    |     self.commit()
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 2917, in __exit__
labelinteractive    |     reraise(new_type, new_type(exc_value, *exc_args), traceback)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 190, in reraise
labelinteractive    |     raise value.with_traceback(tb)
labelinteractive    |   File "/usr/local/lib/python3.7/site-packages/peewee.py", line 3144, in execute_sql
labelinteractive    |     cursor.execute(sql, params or ())
labelinteractive    | peewee.OperationalError: table hastapped has no column named tap_successful
```

Reply:
Quickly tested `db.drop_tables([Label, HasTapped])` out, which seems to work.

But as discussed, will leave it as is for now.